packages/flutter/test/material/switch_test.dart

```diff
@@ -6,41 +6,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-class TestBoxPainter extends BoxPainter {
-  TestBoxPainter(VoidCallback onChanged): super(onChanged);
-
-  @override
-  void paint(Canvas canvas, Offset offset, ImageConfiguration configuration) { }
-}
-
-class TestDecoration extends Decoration {
-  int listeners = 0;
-
-  @override
-  Decoration lerpFrom(Decoration a, double t) {
-    if (t == 0.0)
-      return a;
-    if (t == 1.0)
-      return this;
-    return new TestDecoration();
-  }
-
-  @override
-  Decoration lerpTo(Decoration b, double t) {
-    if (t == 1.0)
-      return b;
-    if (t == 0.0)
-      return this;
-    return new TestDecoration();
-  }
-
-  @override
-  BoxPainter createBoxPainter([VoidCallback onChanged]) {
-    if (onChanged != null)
-      listeners += 1;
-    return new TestBoxPainter(onChanged);
-  }
-}
+import '../services/mocks_for_image_cache.dart';
```

Comment:
I'm amused by this diff. I'm guessing what happened is that we used to have tests back when Switch took a Decoration. Then we changed it to use Images instead, and so we removed the Decoration-based tests (but left the classes, because we forgot to remove them). Then you here remove the old classes since they're not used, and added an import to get ready to add tests to check the images feature, but then didn't add the test. :-)